docs/_articles/en/v3.x/release-information.md

```diff
@@ -5,775 +5,47 @@ lang: en
 
 ### To update the plugin, please follow the linked [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
-## [v2.13.1](https://github.com/forcedotcom/sfdx-scanner/tree/v2.13.1) (03-23-2022)
+## [v3.0.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.0.0) (04-26-2022)
 
-[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.13.0...v2.13.1)
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.0.0...v2.13.1)
```

Comment:
@rmohan20 this link does not work.

Reply:
I just fixed the incorrect diff. Though I don't think the link would work until we create the v3.0.0 tag.